main/lsp/requests/hover.cc

```diff
@@ -37,7 +37,7 @@ LSPResult LSPLoop::handleTextDocumentHover(unique_ptr<core::GlobalState> gs, con
     prodCategoryCounterInc("lsp.messages.processed", "textDocument.hover");
 
     auto result =
-        setupLSPQueryByLoc(move(gs), params.textDocument->uri, *params.position, LSPMethod::TextDocumentHover, false);
+        setupLSPQueryByLoc(move(gs), params.textDocument->uri, *params.position, LSPMethod::TextDocumentHover);
```

Comment:
wouldn't this also make it look like hover works in untyped files?

Reply:
Oof; you're right. I'll fix that.